pkg/app/context.go

```diff
@@ -769,10 +769,20 @@ func (ctx *RequestContext) Reset() {
 	ctx.conn = nil
 }
 
+// Redirect returns an HTTP redirect to the specific location.
+// Note that this will not stop the current handler.
+// in other words, even if Redirect() is called, the remaining handlers will still be executed and cause unexpected result.
+// So it should call Abort to ensure the remaining handlers for this request are not called.
```

Comment:
"handlers for this request" -> "handlers of this request"
&&
"are not called." -> "will not be called."